modules/custom/az_core/az_core.module

```diff
@@ -166,20 +166,6 @@ function az_core_modules_uninstalled($modules) {
   }
 }
 
-/**
- * Implements hook_block_access().
- */
-function az_core_block_access(Block $block, $operation, AccountInterface $account) {
-  if ($operation === 'view' && $block->id() === 'az_barrio_sidebar_menu') {
-    $block_content = \Drupal::entityTypeManager()
-      ->getViewBuilder('block')
-      ->view($block);
-    $markup = \Drupal::service('renderer')->renderRoot($block_content);
-    return AccessResult::forbiddenIf($markup === '')->addCacheableDependency($block);
-  }
-  return AccessResult::neutral();
-}
-
```

Comment:
I think we should try @trackleft's suggestion to restore this hook and have it apply to all menu blocks, not just our specific menu block id:

```
/**
 * Implements hook_block_access().
 */
function az_core_block_access(Block $block, $operation, AccountInterface $account) {
  if ($operation === 'view' && $block->plugin() === 'menu_block') {
    $block_content = \Drupal::entityTypeManager()
      ->getViewBuilder('block')
      ->view($block);
    $markup = \Drupal::service('renderer')->renderRoot($block_content);
    return AccessResult::forbiddenIf($markup === '')->addCacheableDependency($block);
  }
  return AccessResult::neutral();
}
```

Reply:
I think menu_block and az_menu_block should be added

Reply:
Actually, something like this probably 
https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_block_types/az_custom_menu_block/az_custom_menu_block.module#L24-L25